lib/fake_braintree/sinatra_app.rb

```diff
@@ -185,6 +185,14 @@ def hash_from_request_body_with_key(key)
       CreditCard.new(updates, options).update
     end
 
+    # Braintree::PaymentMethod.delete
+    delete '/merchants/:merchant_id/payment_methods/any/:credit_card_token' do
+      cc_hash     = {}
+      options     = {token: params[:credit_card_token], merchant_id: params[:merchant_id]}
```

Comment:
Line is too long. [90/80]<br>Unnecessary spacing detected.<br>Space inside { missing.<br>Space inside } missing.